zuul-groovy/src/test/java/com/netflix/zuul/scriptManager/FilterVerifierTest.java

```diff
@@ -134,21 +131,21 @@ public void testZuulFilterInstance() throws Exception {
         assertNotNull(filterClass);
 
         Object filter2 = FilterVerifier.INSTANCE.instantiateClass(filterClass);
-        assertThrows(InstantiationException.class, () -> FilterVerifier.INSTANCE.checkZuulFilterInstance(filter2));
+        assertThrows(InstantiationException.class,() -> FilterVerifier.INSTANCE.checkZuulFilterInstance(filter2));
     }
 
     @Test
-    public void testVerify() throws Exception {
+    void testVerify() throws Exception {
         FilterInfo filterInfo1 = FilterVerifier.INSTANCE.verifyFilter(sGoodGroovyScriptFilter);
         assertNotNull(filterInfo1);
-        assertEquals(filterInfo1.getFilterID(), "null:filter:in");
-        assertEquals(filterInfo1.getFilterType(), FilterType.INBOUND);
-        assertEquals(filterInfo1.getFilterName(), "filter");
```

Comment:
Not sure if this is the formatter playing from the rewrite tool? Maybe worthwhile running these through the internal formatter to keep things consistent

Reply:
I just pushed a fix for FilterVerifierTest.  OpenRewrite should have preserved whitespace in this file, but it did not.